sourmash/commands.py

```diff
@@ -89,19 +100,40 @@ def compute(args):
                         help="name the signature generated from each file after the first record in the file (default: False)")
     parser.add_argument('--input-is-10x', action='store_true',
                         help="Input is 10x single cell output folder (default: False)")
+    parser.add_argument('--count-valid-reads', default=0, type=int,
```

Comment:
could you update the help message to make it clear that this is about 10x input only?